tests/Qdrant.Client.Tests/PointTests.cs

```diff
@@ -556,5 +556,44 @@ await _client.UpsertAsync("collection_1", new[]
 		Assert.Single(groups, g => g.Hits.Count == 1);
 	}
 
+	[Fact]
+	public async Task Facet()
+	{
+		await CreateAndSeedCollection("collection_1");
+
+		await _client.CreatePayloadIndexAsync(
+			"collection_1",
+			fieldName: "foo",
+			schemaType: PayloadSchemaType.Keyword
+		);
+
+		await _client.FacetAsync(
+			"collection_1",
+			filter: MatchKeyword("bar", "hello"),
+			key: "foo",
+    		limit: 2
+		);
+	}
+
+	[Fact]
+	public async Task SearchMatrix()
+	{
+		await CreateAndSeedCollection("collection_1");
+
+		await _client.SearchMatrixPairsAsync(
+			"collection_1",
+			filter: MatchKeyword("bar", "hello"),
+			sample: 3,
+    		limit: 2
+		);
+
+		await _client.SearchMatrixOffsetsAsync(
+			"collection_1",
+			filter: MatchKeyword("bar", "hello"),
+			sample: 3,
+    		limit: 2
+		);
```

Comment:
should this assert something about the response?

Reply:
This test checks only API validity. I used it to test snippets in PR description which will be included in the documentation. Do you think it's unnecessary?